src/components/ExportButton/ExportButton.js

```diff
@@ -92,23 +100,40 @@ function ExportButton({
           templateValues = getTemplateValues
             ? getTemplateValues(i18n.language, t)
             : {};
-          imageUrl = getOverlayImageUrl && getOverlayImageUrl(i18n.language, t);
+          imageUrl =
+            getOverlayImageUrl &&
+            getOverlayImageUrl(i18n.language, t, exportFormat);
           fileName = getExportFileName
-            ? getExportFileName(i18n.language, t)
+            ? getExportFileName(i18n.language, t, exportFormat)
             : fileName;
         }
 
+        const scaleLineControl = mapToExport
+          .getControls()
+          .getArray()
+          .find((c) => c instanceof ScaleLine);
+        const scaleLineElement =
+          scaleLineControl &&
+          getStyledPdfScaleLine(scaleLineControl, exportSelection?.resolution);
+        const scaleLineCanvas =
+          scaleLineElement && (await html2canvas(scaleLineElement));
```

Comment:
Move this to an utils function so we can add a test.
const scaleLineCanvas =  getScaleLineCanvas(map, resolution)